ui/packages/catalog-ui-search/src/main/webapp/component/search-form-editor/search-form-editor.view.js

```diff
@@ -118,10 +123,50 @@ module.exports = Marionette.LayoutView.extend({
         selectionInterface: new SelectionInterface(),
         onMapLoaded: olMap => {
           this.showQueryForm(collection, id)
+          if (this.model.get('filterTemplate')) {
```

Comment:
✏️ I would recommend pulling the model attribute into a variable since it's being referenced multiple times here:

```
const filterTemplate = this.model.get('filterTemplate')
```
also reversing the predicate would collapse the nesting nicely:
```
if (!filterTemplate)
  return
<normal code flow without '{}'>
```